src/saveload/saveload.h

```diff
@@ -331,6 +331,7 @@ enum SaveLoadVersion : uint16_t {
 	SLV_CUSTOM_SUBSIDY_DURATION,            ///< 292  PR#9081 Configurable subsidy duration.
 	SLV_SAVELOAD_LIST_LENGTH,               ///< 293  PR#9374 Consistency in list length with SL_STRUCT / SL_STRUCTLIST / SL_DEQUE / SL_REFLIST.
 	SLV_RIFF_TO_ARRAY,                      ///< 294  PR#9375 Changed many CH_RIFF chunks to CH_ARRAY chunks.
+	SLV_AUTOMATIC_SEPARATION,               ///< 295  PR#8342 Allow automatically separating vehicles in shared orders.
```

Comment:
This is rebased incorrectly. It needs to be the last entry in the list (324 as I write this), just above SL_MAX_VERSION.

Reply:
Whoops, not sure how I missed this. Fixed. :)